### PR TITLE
W-8033900: Updated Readme to add security information.

### PR DESCRIPTION
**Pull request by lori-adams**
I added some security information to the Readme based on material provided by Yassir Elley and Austin Siford.  Please review and merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,10 +286,21 @@ DEBUG level logging enables logging of:
...
+For more information about security considerations related to developing headless commerce applications, see 
+[Security Considerations for Headless Commerce](https://developer.commercecloud.com/s/article/HeadlessSecurity) on the 
+[Commerce Cloud Developer Center](https://developer.commercecloud.com).
+
+If you discover any potential security issues, please report them to security@salesforce.com as soon as possible.
+
+
 ## Additional Documentation
 
-- [Examples](./examples)
```
Comment (bredmond-sf) [inline README.md:291]: Looks like this commit accidentally removes the Examples link that was recently added.

Comment (lori-adams) [reply README.md:291]: Oh yes, good catch!

Comment (unandyala) [reply README.md:291]: please add relative path to the examples (./examples)

Comment (monkey-jeff) [reply README.md:291]: > please add relative path to the examples (./examples)

Actually, please provide absolute paths so they won't be broken when deployed to NPM.  But you don't have to fix that. We should.

File: README.md [hunk 2]
```diff
@@ -286,11 +286,24 @@ DEBUG level logging enables logging of:
 
 **Note:** Debug level logging may expose sensitive data in the logs
 
+## Security
+This library does not maintain or manage authentication tokens. The consumer of the SDK is responsible for writing code to retrieve, store, and refresh authentication tokens.
```
Comment (monkey-jeff) [inline README.md:290]: The SDK does provide retrieval, just not store and refresh.  Does that matter?

Comment (lori-adams) [reply README.md:290]: Let me check with Yassir Elley and Austin Siford, who provided that text to me.

**Review by bredmond-sf — APPROVED** [review]

**Review by unandyala — APPROVED** [review]

**Comment by lori-adams** [comment]
Created a new request with signed commit.

**Pull request closed without merging**